tests/integration/src/test/java/org/apache/pulsar/tests/integration/functions/PulsarStateTest.java

```diff
@@ -57,7 +57,8 @@ public class PulsarStateTest extends PulsarStandaloneTestSuite {
 
     public static final String WORDCOUNT_PYTHON_FILE = "wordcount_function.py";
 
-    @Test(groups = {"python_state", "state", "function", "python_function"})
+    // skip this tests and fix it later
+//    @Test(groups = {"python_state", "state", "function", "python_function"})
```

Comment:
Is this also required with `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python` ?

Reply:
Yes. The function tests using python runtime will meet the same exception. I added the variable in the docker, and I will revert this change.